tests/micro/qemu/zephyr-runtime/src/main.c

```diff
@@ -161,6 +162,25 @@ tvm_crt_error_t TVMPlatformTimerStop(double* elapsed_time_seconds) {
   return kTvmErrorNoError;
 }
 
+tvm_crt_error_t TVMPlatformGenerateRandom(uint8_t* buffer, size_t num_bytes) {
+  while (num_bytes > 0) {
```

Comment:
Since the loop should run `num_bytes / sizeof(random)` iterations, I think it's better to use for-loop instead of using while-loop paired with continue statement.

Reply:
rewritten to use for-loop!